src/components/radio/RadioSelectActions.ts

```diff
@@ -0,0 +1,27 @@
+import {IReduxAction} from '../../utils/ReduxUtils';
+
+export interface IRadioSelectActionPayload {
+    id: string;
+    value?: string;
+    disabledValues?: string[];
+}
+
+export const RadioSelectActions = {

```

Comment:
Suggestion: rename this const `RadioSelectActionTypes`

Then instead of exporting the actions individually, export the actions at the end of the file like the following:
```ts
export const RadioSelectActions = {
    set: setRadioSelect,
    remove: removeRadioSelect,
};
```
or even better rename the actions so you can do this `export const RadioSelectActions = {set, remove};`.

I know this is not how we typically export actions in this repo, and this is only a suggestion to avoid exporting too much things. I have a personnal preference for this, because it allows to reuse the same action names accross components.

Reply:
you should suggest this in a jsadmin update and see if the group agrees with that, i'll leave it as it is for now